### PR TITLE
adding mongo updates to the abbey script

### PR DESCRIPTION
**Pull request by jarv**

### CODE REVIEW

File: util/vpc-tools/abbey.py [hunk 1]
```diff
@@ -1,9 +1,10 @@
-#!/usr/bin/env python -u
```
Comment (feanil) [inline util/vpc-tools/abbey.py]: nitpick: blank first line

File: util/vpc-tools/abbey.py [hunk 2]
```diff
@@ -14,12 +14,76 @@
...
 EC2_RUN_TIMEOUT = 180  # time to wait for ec2 state transition
 EC2_STATUS_TIMEOUT = 300  # time to wait for ec2 system status checks
 NUM_TASKS = 5  # number of tasks for time summary report
 
 
+class MongoConnection:
+
+    def __init__(self):
+        try:
+            mongo = MongoClient(host=args.mongo_host, port=args.mongo_port)
```
Comment (feanil) [inline util/vpc-tools/abbey.py:31]: did you mean to pass in the `args` variable to the `__init__` function?  How are you getting that info?

Comment (jarv) [inline util/vpc-tools/abbey.py:31]: args is global

**Comment by feanil** [comment]
Were you planning on adding logic so  the splunkforwarder and datadog plays are run on all instances as part of this PR?

**Comment by jarv** [comment]
would like to hold off on that for now since the plays don't exist yet

**Comment by feanil** [comment]
Plays were merged in yesterday.  https://github.com/edx/configuration/blob/master/playbooks/edx-east/datadog.yml and https://github.com/edx/configuration/blob/master/playbooks/edx-east/splunkforwarder.yml

**Comment by feanil** [comment]
looks good.

**Pull request merged**